src/utils/windowUtils.ts

```diff
@@ -41,6 +41,34 @@ export module Utils {
       }
     }
 
+    /**
+     * Sends a deprecation warning to the console. The warning includes the version number of the deprecation,
+     * the name of the function which was deprecated and an optional message.
+     *
+     * To be used in the first line of a deprecated method.
+     *
+     * @param {string} version The version when the tagged method became obsolete
+     * @param {string?} message Optional message to be shown with the warning
+     */
+    export function deprecated(version: string, message = "") {
```

Comment:
Survey where? Is this an important thing? It is supposed to only be used by developers.